components/authz-service/storage/v2/storage.go

```diff
@@ -234,6 +234,7 @@ func DefaultRoles() []Role {
 			"ingest:*",
 			"secrets:*",
 			"telemetry:*",
+			"applications:*",
```

Comment:
This is incompatible with 45_add_application_policies.up.sql and will result in the entry being added to the database twice. Just delete this line and all is well.

Same for L258 below.